commands/station.js

```diff
@@ -30,21 +30,19 @@ const station = async ({ json, experimental }) => {
     await fs.mkdir(join(paths.moduleState, moduleName), { recursive: true })
   }
 
-  activities.on('activity', activity => {
+  activities.onActivity(activity => {
     if (json) {
       console.log(JSON.stringify({
         type: `activity:${activity.type}`,
-        timestamp: activity.timestamp,
         module: activity.source,
-        message: activity.message,
-        id: activity.id
```

Comment:
Since we don't persist to disk any more, the consumer (Station Desktop for example) can just attach `timestamp` and `id` while recording the events. Core doesn't need to care about these logistics I think